### ISSUE TITLE
Query comments on the blog-post template as a page query 💬

### ISSUE THREAD

**Issue by robinmetral** [robinmetral/eaudepoisson#20]
Instead of the current `<StaticQuery>` on `Comments.js`.

This will allow for better filtering *within* graphql (instead of using `.filter()`, and let me display a comment count in the article's header.

**Comment by robinmetral** [comment]
This is my query:

```graphql
  allCommentsYaml(filter: {slug: {eq: "/de-vientiane-a-bangkok-par-rail-et-route/"}}) {
    edges {
      node {
        slug
        date
        name
        message
      }
    }
  }
```

How to make the filter **dynamic**?

i.e. on blog post X, query comments with slug X, etc.

**Comment by robinmetral** [comment]
Fixed by #24
